addons/azurecommentsanalytics-1.0.0/chart/azure-comments-analytics/templates/_helpers.tpl

```diff
@@ -12,4 +12,8 @@
 
 {{- define "azure-comments-analytics.workspace" -}}
 {{- .Values.workspaceName -}}
+{{- end -}}
+
+{{- define "azure-comments-analytics.channel" -}}
```

Comment:
This definition is unnecessary. You don't use it anywhere.

Reply:
Removed it